### PR TITLE
Initial commit of Vets.gov Prescription API

### PR DESCRIPTION
**Pull request by saneshark**

### CODE REVIEW

File: lib/va_rx/api/prescriptions.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+module VARx
+  module API
+    # This module defines the prescription actions
+    module Prescriptions
+      def get_active_rx(*args)
+        perform_with_objects(:get, 'prescription/getactiverx', args.pop, Prescription)
+      end
+
+      def get_history_rx(*args)
+        perform_with_objects(:get, 'prescription/gethistoryrx', args.pop, Prescription)
```
Comment (cfeeney-va) [inline lib/va_rx/api/prescriptions.rb]: might be a newbie question, but where do we get "perform_with_objects" from? is it inherited from somewhere, or is defining it a to-do?

File: lib/va_rx/error_codes.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      MISSING_USER_CREDENTIALS = 104
+      USER_NOT_FOUND = 105
+      USER_NOT_ELIGIBLE = 106
+      UNABLE_TO_CREATE_TOKEN = 107
+      MISSING_SESSION_TOKEN = 108
+      INVALID_SESSION_TOKEN = 109
+      EXPIRED_SESSION_TOKEN = 110
+      INVALID_USER_PERMISSIONS = 111
+      DATA_INTEGRITY_ERROR = 117
+      MISSING_APPLICATION_TOKEN = 108
```
Comment (cfeeney-va) [inline lib/va_rx/error_codes.rb]: should be 132 (instead of 108)

**Comment by cfeeney-va** [comment]
looks great!!!  can you update the one error code i flagged?

File: lib/va_rx/api/prescriptions.rb (hunk 1 above)
Comment (saneshark) [inline lib/va_rx/api/prescriptions.rb]: It's a great question and I probably should have put a TODO: for that. I haven't written the method yet because basically it's going to be a wrapper for the Faraday request object. I'm still kind of thinking through how it will work for the one Post request (whether it should receive an object for consistency or just a simple hash -- leaning towards object for consistency).

TLDR; The perform_with_objects and perform_with_object (singular) haven't been implemented yet. They'll be part of the Client class and I plan on having that pushed today.

**Comment by cfeeney-va** [comment]
LGTM
